packages/react-native-windows-init/src/Cli.ts

```diff
@@ -83,14 +83,14 @@ export const windowsInitOptions = initOptions({
     type: 'string',
     describe: 'The type of project to initialize (supported on 0.64+)',
     choices: ['app', 'lib'],
-    default: 'app',
+    default: undefined,
```

Comment:
I think for back-compat and test validation, it makes more sense here to leave this value as set, and up[date the description to include the range of versions, like `supported on  0.64-0.70` or whatever.